railgun/internal/proxy/clientgo_cached_proxy_resolver_test.go

```diff
@@ -133,7 +133,7 @@ func TestCaching(t *testing.T) {
 
 	t.Log("flushing the cache state to kong admin api")
 	previousUpdateCount := fakeKongAdminUpdateCount()
-	proxy.syncTicker.Reset(time.Millisecond * 200)
+	proxy.syncTicker.Reset(time.Millisecond * 400)
```

Comment:
Q: what is the context of this change, can you please help me to better understand what this fixes?

Reply:
The clientgo_cached_proxy_resolver_test is not unstable either in CI or my local test environment. 
Extend the sync ticker to cover corner case which takes more time sync.